sdk/core/System.ClientModel/tests/Pipeline/ClientLoggingPolicyTests.cs

```diff
@@ -0,0 +1,292 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Options;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using ClientModel.Tests;
+using ClientModel.Tests.Mocks;
+using NUnit.Framework;
+using SyncAsyncTestBase = ClientModel.Tests.SyncAsyncTestBase;
+
+namespace System.ClientModel.Tests.Pipeline;
+
+public class ClientLoggingPolicyTests : SyncAsyncTestBase
```

Comment:
I don't see any tests of ClientLoggingPolicy alone here, only through derived types.  It is good to have tests for the cases we want to validate derived type support for.  Are these the main scenarios?  I had understood the default pipeline for a third-party client would hold an instance of ClientLoggingPolicy itself -- would it make sense to add tests for those case here?  (Or all they all represented in the ClientPipelineFunctionalTests?)

Reply:
I agree it would make sense to add some tests for ClientLoggingPolicy alone here to cover those scenarios